src/modules/mod_keybinds/index.js

```diff
@@ -0,0 +1,55 @@
+const $ = require('jquery');
+const twitch = require('../../utils/twitch');
+const keyCodes = require('../../utils/keycodes');
+const chatModerationCards = require('../chat_moderator_cards');
+
+const CHAT_TEXT_AREA = '.ember-chat .chat-interface textarea';
+
+class ModKeybindsModule {
+    constructor() {
+        this.load();
+    }
+
+    load() {
+        $('body').on('keydown', e => this.onKeyDown(e));
```

Comment:
I don't want these attached to body.. also, can we just make a separate file under the mod card for keybinds, and then import and run this when a modcard is created? extending its class for current card info seems out of place.